docs/user/setup.rst

```diff
@@ -93,16 +93,21 @@ installation of Arch Linux.
     $ sudo dnf groupinstall "Development Tools"
     $ sudo dnf install gc-devel zlib-devel # both optional
 
-**FreeBSD 12.2 and later**
+**FreeBSD 12.4 and later**
 
 .. code-block:: shell
 
-    $ pkg install llvm10
+    $ pkg install llvm # optional
```

Comment:
Perhaps remove line entirely if acceptable version of 
llvm comes with FreeBSD 12.4?  Reduce cognitive load.